domain/src/main/java/woowacourse/movie/policy/decorator/DatePolicy.kt

```diff
@@ -0,0 +1,21 @@
+package woowacourse.movie.policy.decorator
+
+import woowacourse.movie.policy.Policy
+
+class DatePolicy(policy: Policy) : PolicyDecorator(policy) {
+    override fun cost(): Int {
+        var payment = payment
+        payment = movieDayDiscount(payment)
+        return payment
```

Comment:
```suggestion
        return movieDayDiscount(payment)
```
굳이 여러번 payment 를 선언해줄 필요가 있을까요? payment 값을 추적하기가 매우 어려울 것 같아요.